plugins/main/windows/win-bridge/win-bridge_windows.go

```diff
@@ -215,5 +215,5 @@ func cmdCheck(_ *skel.CmdArgs) error {
 }
 
 func main() {
-	skel.PluginMain(cmdAdd, cmdCheck, cmdDel, version.PluginSupports("0.1.0", "0.2.0", "0.3.0"), bv.BuildString("win-bridge"))
+	skel.PluginMain(cmdAdd, cmdCheck, cmdDel, version.PluginSupports("0.1.0", "0.2.0", "0.3.0", version.Current()), bv.BuildString("win-bridge"))
```

Comment:
This should have "0.4.0" in it as well.

Reply:
@squeed @AkihiroSuda actually this should be `version.All`

Reply:
Nevermind, this comment is wrong.